### ISSUE TITLE
(Help Wanted) Merging PS2 branch into main branch

### ISSUE THREAD

**Issue by LlysiX** [hmxmilohax/rock-band-2-deluxe#29]
The [ps2-merge-work](https://github.com/hmxmilohax/rock-band-2-deluxe/tree/ps2-merge-work) branch is designed to make the process of merging these platforms together as easy as possible, using the main branch as a base and adding/modifying files so that the main files match the ps2 files.

Differing dta files can be changed using `#ifdef HX_EE`, `#else`, and `#endif` for ps2 specific items, or `#ifndef HX_EE`, `#else`, and `#endif` for non-ps2 items.

The best way to compare and help make changes to this branch is to use [WinMerge](https://winmerge.org) (or similar software) to compare the _ark folder of the [ps2 branch](https://github.com/hmxmilohax/rock-band-2-deluxe/tree/ps2) to the same folder in ps2-merge-work, making changes to the dtas in ps2-merge-work so that the conflicting points match. Files that are only found in the _ark folder on the ps2 branch should be placed in the _ps2 folder.

This issue is here to help document progress. Changed files, incomplete files, etc.

Until all files have been successfully merged, test on ps3 and 360 to make sure the game still boots on those.
Once all files have been merged, begin testing on pcsx2 and make adjustments until there are no TLB Misses reported in the log.
Finally, test on PS2 Hardware to see if the game will successfully boot.

**Comment by LlysiX** [comment]
Status update: all ps2 only files have now been placed in the _ps2 folder.
drum_trainer_menu.dta successfully merged.

**Comment by LlysiX** [comment]
added more files to _ps2, any files in there do not need to be merged from ps2 branch _ark folder.

**Comment by LlysiX** [comment]
### ALL DTAS THAT NEED TO BE CONVERTED:
![image](https://user-images.githubusercontent.com/79173634/230502945-b3f552bd-f6b6-4e60-9ac4-ac32a9a0848b.png)
![image](https://user-images.githubusercontent.com/79173634/230502959-761dd7bb-f91b-45ea-af80-18154aff68e2.png)
![image](https://user-images.githubusercontent.com/79173634/230502977-8838eaee-4b1f-477f-96af-543bcd154b29.png)

**Comment by LlysiX** [comment]
I made an update merging band, beatmatch_controller, and long_cheats. 
From here on out, the status should be checked in the [commit history of the ps2 merge branch](https://github.com/hmxmilohax/rock-band-2-deluxe/commits/ps2-merge-work) and not this issue, as it would be a lot of comments if I reported every little update I make.
Basically, if you want to contribute, use the [commit history](https://github.com/hmxmilohax/rock-band-2-deluxe/commits/ps2-merge-work) to see what is done, and make your commit the names of the files you have merged.
